app/models/user_intervention.rb

```diff
@@ -5,6 +5,7 @@ class UserIntervention < ApplicationRecord
   belongs_to :user, inverse_of: :user_interventions
   belongs_to :intervention, inverse_of: :user_interventions
   has_many :user_sessions, dependent: :destroy
+  has_many :phone_answers, through: :user_sessions, class_name: 'Answer::Phone', source: :answers
```

Comment:
why?

Reply:
We have a requirement to support user defined time ranges for smses in classic sessions. I have found, that we are storing user defined time ranges only in Answers. This is easiest way to find all such responses and base on them.